auth/src/test/java/com/redhat/lightblue/rest/auth/ldap/LightblueLdapRoleProviderTest.java

```diff
@@ -1,252 +1,117 @@
+    private static final String USER_WITH_ROLES = "uid=derek63,ou=Users,dc=example,dc=com";
+    private static final String USER_WITH_NO_ROLES = "uid=lcestari,ou=Users,dc=example,dc=com";
+    private static final String BASEDB_GROUPS = "ou=Departments,dc=example,dc=com";
 
+    private static LightblueLdapRoleProvider provider;
 
-        ldapServer = new LdapServer();
-        ldapServer.setTransports(new TcpTransport(11389));
-        ldapServer.setDirectoryService(directoryService);
-
-        directoryService.startup();
-        ldapServer.start();
-    }
-
-    @AfterClass
-    public static void stopApacheDs() throws Exception {
-        ldapServer.stop();
-        directoryService.shutdown();
-        directoryService.getWorkingDirectory().delete();
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        ldapServer.add(BASEDB_USERS, new Attribute[] { new Attribute("objectClass", "top"), new Attribute("objectClass", "organizationalUnit"),
+                new Attribute("ou", "Users") });
+        ldapServer.add(BASEDB_GROUPS, new Attribute[] { new Attribute("objectClass", "top"), new Attribute("objectClass", "organizationalUnit"),
+                new Attribute("ou", "Groups") });
+        ldapServer.add(USER_WITH_NO_ROLES, new Attribute[] { new Attribute("cn", "lcestari"), new Attribute("uid", "lcestari"),
+                new Attribute("objectClass", "person") });
+        ldapServer.add(USER_WITH_ROLES, new Attribute[] { new Attribute("cn", "derek63"), new Attribute("uid", "derek63"),
+                new Attribute("memberOf", "cn=lightblue-contributors,ou=Groups,dc=example,dc=com"),
+                new Attribute("memberOf", "cn=lightblue-developers,ou=Groups,dc=example,dc=com"), new Attribute("objectClass", "person") });
+        
+        System.setProperty("ldap.host", "localhost");
+        System.setProperty("ldap.port", String.valueOf(LdapServerExternalResource.DEFAULT_PORT));
+        System.setProperty("ldap.database", "test");
+        System.setProperty("ldap.person.basedn", BASEDB_USERS);
+        System.setProperty("ldap.department.basedn", BASEDB_GROUPS);
+
+        provider = new LightblueLdapRoleProvider(
+                "ldap://localhost:" + LdapServerExternalResource.DEFAULT_PORT, 
+                LdapServerExternalResource.DEFAULT_BASE_DN,
+                LdapServerExternalResource.DEFAULT_BINDABLE_DN, 
+                LdapServerExternalResource.DEFAULT_PASSWORD);
     }
 
-    @Test
-    public void testGetUserRolesInvalidLDAPDirectory() throws Exception {
-        LightblueLdapRoleProvider provider = new LightblueLdapRoleProvider(LDAP_LOCALHOST_11389, "schema", null, null);
-
-        List<String> userRoles = provider.getUserRoles("nameHere");
-        assertNotNull(userRoles);
-        assertEquals("getUserRoles method should return an empty list",0, userRoles.size());
+    public LightblueLdapRoleProviderTest() throws Exception {
+        super();
     }
 
     @Test
     @Ignore
```

Comment:
Actually I meant to take out testInMemoryLdapServer altogether, as it doesn't test anything in LightblueLdapRoleProvider.  Removing now